BUILD

```diff
@@ -22,3 +22,31 @@ remote_environment(
     name="buildgrid_remote",
     python_bootstrap_search_path=["<PATH>"],
 )
+
+files(
+    name="all-__init__.py-files",
+    sources=[
+        "**/__init__.py",
+        "!testprojects/**",
+        # These are explicit namespace packages
+        "!src/python/pants/__init__.py",
+        "!src/python/pants/testutil/__init__.py",
+    ],
+)
+
+# NB: This should be in `lint` when we implement `lint` in https://github.com/pantsbuild/pants/issues/17729
+experimental_test_shell_command(
```

Comment:
I know nothing about this. How do you invoke it?

Reply:
It gets executed as part of the `test` goal! :tada: